### PR TITLE
navbar menu on mobile version

### PR DESCRIPTION
**Pull request by ybadr99**
- Add navbar menu 
- using JS to show navbar menu on click

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  document.querySelector('header').classList.toggle('fixed');
+});
+
+navItem.forEach((n) => {
+  n.addEventListener('click', () => {
+    menuIcon.classList.remove('active');
+    navMenu.classList.remove('active');
+    logo.classList.remove('hide');
+  });
+});
```
Comment (Qoosim) [inline src/script.js]: - [ ] Wonderful job so far, however, the behaviour of your mobile menu is not consistent. At first page load, everything seems fine, the hamburger brings out the mobile menu when it is clicked. But sometimes, the page does not scroll to the appropriate section of the page when any of the menu items is clicked.  Please, take a look. :point_down: 

[mobile-menu-behaviour.webm](https://user-images.githubusercontent.com/26597432/217362603-2929d05d-05e4-45b1-9bd4-7cd4d093b8e6.webm)

[mobile-menu-2.webm](https://user-images.githubusercontent.com/26597432/217362652-f4adf4d4-cc4b-4303-813b-5c7267c7176e.webm)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️

- [ ] The MIT license of your repo is not properly set. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/script.js (hunk 1 above)
Comment (ybadr99) [reply src/script.js]: hey 
thanks for the good review 
now, I know the issue and gonna to fix it

Comment (ybadr99) [reply src/script.js]: I had pushed MIT license on the main branch

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @ybadr99, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ] Great effort working on the changes requested by the previous reviewer, but currently the MIT license link is still not fixed yet as the link is still broken. Kindly fix it
> N.B: If you need help to get this fixed kindly reach out and I would be glad to render a helping hand.

MIT image 👇 

mit


- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ybadr99** [comment]
Hi @brainconnect93 
I'm already fixed it in the main branch 
Screenshot from 2023-02-08 00-48-55

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @ybadr99, 👋 

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 



## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️

### Changes Required ♻️

- [x]  _Please check the comments under the review._

### Optional Changes 🔥

Thank You.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**